packages/ui-components/src/components/FavouriteButton.tsx

```diff
@@ -7,9 +7,19 @@ import FavoriteBorderIcon from '@material-ui/icons/FavoriteBorder';
 import FavoriteIcon from '@material-ui/icons/Favorite';
 import IconButton from '@material-ui/core/IconButton';
 
-import PropTypes from 'prop-types';
+interface FavouriteButtonProps {
+  isFavourite?: boolean;
+  onChange: () => void;
+  color?: 'primary' | 'secondary' | 'inherit' | 'default';
```

Comment:
Can these be pulled from material ui? It seems so since they are implicitly used in the IconButton.tsx component below this?